package.json

```diff
@@ -21,6 +21,8 @@
     "amo:ui-tests": "better-npm-run amo:ui-tests",
     "amo:no-proxy": "better-npm-run amo:no-proxy",
     "amo:stage": "better-npm-run amo:stage",
+    "build-storybook": "better-npm-run storybook-build",
+    "deploy-storybook": "storybook-to-ghpages -- --remote=upstream",
```

Comment:
I did to but I couldn't get it to work (tested locally) unless is matched their names..

Reply:
regarding the remote

The default remote is origin...will that be ok then?

Reply:
> I did to but I couldn't get it to work (tested locally) unless is matched their names..

mmh, not sure I understand.

> The default remote is origin...will that be ok then?

I don't know, do we want to use it manually? If not, then it should the name of the remote that the automated process will use.

Reply:
I don't think we want to do this manually. I guess we can remove the remote info and use something like this ~:

//travis.yml
after_success: yarn deploy-storybook -- --ci > /dev/null 2>&1 

and then it will know the remote? 

storybook-deployer ci REF: https://github.com/storybooks/storybook-deployer#deploying-storybook-as-part-of-a-ci-service

Reply:
regarding the script name. initially this wasn't working (it was complaining about the config set up (that it was in their default)) but now it seems ok 🤷‍♀️

Reply:
I will update..